marimbabot_ur5_flex_double_moveit_config/config/marimbabot_ur5.srdf

```diff
@@ -96,15 +96,6 @@
         <joint name="ur5_wrist_2_joint" value="1.6173"/>
         <joint name="ur5_wrist_3_joint" value="0"/>
     </group_state>
-    <group_state name="marimbabot_home" group="arm_mallets">
```

Comment:
Why did you delete the home position? It is utilized at multiple points in the planning pipeline?

Reply:
two marimbabot_home was defind and looked like a duplicate. but just saw that one line is different "mallet_finger" in second marimbabot_home group. I think we should delete the first defined group.

Reply:
We can keep both as they are configurations for different groups. One is for "arm_mallets" and includes a state for the `mallet_finger` and one is only for the `arm` (the old single mallet setup). We can keep both as they do not result in any problems and define a homing position for both groups.

Reply:
I would revert [8997cc3](https://github.com/UHHRobotics22-23/marimbabot/pull/268/commits/8997cc3b00e4c0c0dedd05830bb06b81417c6b18)